### PR TITLE
Page content nf

### PR DESCRIPTION
**Pull request by NickFoden**

### CODE REVIEW

**Comment by NickFoden** [comment]
Making this ready for review. Still WIP though. Just have to shift gears. Sso if we merge and someone else picks it up that might make sense. I can work some more on this a bit later on.

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+import socialMeetup from "../assets/social-meetup.svg";
+import socialTwitter from "../assets/social-twitter.svg";
+
+export default () => {
+  return (
+    <div className="footer-container">
```
Comment (matthew-gerstman) [inline src/components/Footer.tsx]: we should use semantic HTML so `<footer>`

File: src/css/styles.css [hunk 2]
```diff
@@ -75,6 +75,24 @@ a:hover {
   color: var(--color-hyperlink-hover);
 }
 
+.footer-container {
+  align-items: center;
+  justify-content: center;
+  background: linear-gradient(to right,#61DAFB, #618EFB);
```
Comment (matthew-gerstman) [inline src/css/styles.css]: can you use color variables here?

File: src/css/styles.css [hunk 3]
```diff
@@ -116,19 +151,26 @@ a:hover {
...
+  }
+}
+@media only screen and (max-width: 767px) {}
 @media only screen and (min-width: 768px) and (max-width: 991px) {
   body {
     font-size: 18px;
     line-height: 1.5;
   }
 
+  .subhead,
```
Comment (matthew-gerstman) [inline src/css/styles.css:179]: just curious, is this based on some style guide? i dont have a strong opinion about the order

File: src/css/styles.css [hunk 4]
```diff
@@ -144,7 +186,25 @@ a:hover {
...
+
   .subhead {
     font-size: 22px;
   }
+
+  .text-group {
+    padding: 0 70px;
+  }
+}
+@media only screen and (max-width: 767px) {
```
Comment (matthew-gerstman) [inline src/css/styles.css:215]: We should probably make these variables somewhere

**Review by matthew-gerstman — APPROVED** [review]
This looks great overall. Feel free to merge with the base branch. None of the comments are blocking but can you leave TODOs if you wont get to them?

File: src/css/styles.css (hunk 3 above)
Comment (NickFoden) [reply src/css/styles.css:179]: I believe this is prettier at work. I didn't reorder, but i like it.

File: src/css/styles.css (hunk 4 above)
Comment (NickFoden) [reply src/css/styles.css:215]: @matthew-gerstman what is best way to leave a todo. I would guess we can import from the `responsive.css` file, but i haven't taken stab at that yet.

Comment (matthew-gerstman) [inline src/css/styles.css:215]:
```suggestion
/* TODO: Standardize how we handle media queries in CSS */
@media only screen and (max-width: 767px) {
```

**Review by matthew-gerstman — COMMENTED** [review]
Add suggestion

**Pull request merged**